src/database/models/index.js

```diff
@@ -1,12 +1,12 @@
-/* eslint-disable */
+"use strict";
```

Comment:
What's the use of this line?

Reply:
normally it says that the code should be executed in strict mode and also it was generated automatically when creating a new model